content/lxc/getting-started.md

```diff
@@ -399,10 +399,15 @@ Next up is `/etc/lxc/lxc-usernet` which is used to set network devices quota for
 
 This means that "your-username" is allowed to create up to 10 veth devices connected to the lxcbr0 bridge.
 
+Create the `~/.local/share/lxc` directory if it doesn't exist and ensure `~/.local` is executable.
+
+    mkdir -p ~/.local/share/lxc && chmod -R +x ~/.local
+
 With that done, the last step is to create an LXC configuration file.
 
 * Create the `~/.config/lxc` directory if it doesn't exist.
 * Copy `/etc/lxc/default.conf` to `~/.config/lxc/default.conf`
+* Set `lxc.apparmor.profile = lxc-container-default-cgns`
```

Comment:
This allows things we should not allow by default, yes modern systemd needs it in some cases but that's still not a good idea to have it recommended by default.

Reply:
It's clear that copying the default config as recommended has `generate` set which fails for unprivileged containers started by a user. Considering this part of the guide is for that use case, what would be recommend default if the current recommendation fails?

Reply:
`lxc-container-default` is the safe profile

Reply:
Okay. I guess that is confusing given the manpages saying

> The default is lxc-container-default-cgns if the host kernel is cgroup namespace aware, or lxc-container-default otherwise.

Reply:
> `lxc-container-default` is the safe profile

Is `lxc-container-default` only recommended for non cgroup systems? Is there ever a time when `lxc-container-default` is appropriate for cgroup v2 systems? Can you be a little more explicit about these safety problems you are referring to?